src/DynamoWebServer/Messages/MessageHandler.cs

```diff
@@ -528,7 +540,7 @@ private string GetInOutPortsData(NodeModel node)
             stringBuilder.Append(inPorts.Any() ? inPorts.Aggregate((i, j) => i + "," + j) : "");
             stringBuilder.Append("], \"OutPorts\": [");
             stringBuilder.Append(outPorts.Any() ? outPorts.Aggregate((i, j) => i + "," + j) : "");
-            stringBuilder.Append("], \"Data\": \"" + GetValue(node) + "\"}");
+            stringBuilder.Append("]}");
```

Comment:
@Benglin, since a code block is being updated in a separate response we don't need in/out ports data in `data` computation response. CodeBlockDataResponse don't need value data